### DISCUSSION TITLE
关于频繁提示已获取root の讨论

### DISCUSSION THREAD

**Discussion by YueXuan-77** [Mygod/VPNHotspot#560] [Q&A]
没过几分钟，麦吉斯克就向我弹出VPN热点已获取root权限（已经在很多版本的面具测试过，都是一样），为什么权限需要这么频繁？在我使用其他应用时，也会出现VPN热点获取了root权限的通知。其他获取root的应用并不会这样。是我的操作问题，还是bug呢？

Within a few minutes, magisk popped up to me that the VPN hotspot had obtained root privileges (tested on many versions of the mask, all the same), why do permissions need to be so frequent? When I use other apps, I also get notifications that the VPN hotspot has gained root privileges. Other apps that get root don't. Is it my operation, or is it a bug?

**Comment by Mygod** [comment]
Root daemon is shutdown if it is idle or if system is low in memory.
